content/en/docs/concepts/instrumentation/_index.md

```diff
@@ -12,32 +12,41 @@ from the system's components must emit [traces](/docs/concepts/signals/traces/),
 [metrics](/docs/concepts/signals/metrics/), and
 [logs](/docs/concepts/signals/logs/).
 
-Without being required to modify the source code you can collect telemetry from
-an application using [automatic instrumentation](automatic/). If you previously
-used an APM agent to extract telemetry from your application, Automatic
-Instrumentation will give you a similar out of the box experience.
-
-To facilitate the instrumentation of applications even more, you can
-[manually instrument](manual/) your applications by coding against the
-OpenTelemetry APIs.
-
-For that you don't need to instrument all the dependencies used in your
-application:
-
-- some of your libraries will be observable out of the box by calling the
-  OpenTelemetry API themselves directly. Those libraries are sometimes called
-  **natively instrumented**.
-- for libraries without such an integration the OpenTelemetry projects provide
-  language specific [Instrumentation Libraries][]
-
-Note, that for most languages it is possible to use both manual and automatic
-instrumentation at the same time: Automatic Instrumentation will allow you to
-gain insights into your application quickly and manual instrumentation will
-enable you to embed granular observability into your code.
-
-The exact installation mechanism for [manual](manual/) and
-[automatic](automatic/) instrumentation varies based on the language you’re
-developing in, but there are some similarities covered in the sections below.
-
-[instrumentation libraries]:
-  /docs/specs/otel/overview/#instrumentation-libraries
+OpenTelemetry provides [APIs and SDKs for eleven languages](/docs/languages) to
+facilitate the process of instrumenting your code. Among others it provides the
+following benefits:
+
+- Each language-specific implementation of the API and SDK follows the
+  requirements and expectations of the
+  [OpenTelemetry specification](/docs/specs/otel/)
+- Libraries can leverage the OpenTelemetry API as a dependency, which will have
+  no impact on applications using that library, unless the OpenTelemetry SDK is
+  imported.
+- For each [signal](/docs/concepts/signals) (traces, metrics, logs) you will
+  find methods to create them, process them and export them.
+- With [context propagation](/docs/concepts/context-propagation) being built
+  into the implementations, you can correlate signals, regardless of where they
+  are generated
+- [Resources](/docs/concepts/resources) and
+  [Instrumentation Scopes](/docs/concepts/instrumentation-scope) allow grouping
+  of signals, by different entities.
```

Comment:
```suggestion
- [Resources](/docs/concepts/resources) and
  [Instrumentation Scopes](/docs/concepts/instrumentation-scope) let you group
  signals by different entities.
```

Reply:
Can we add an example?

Reply:
I was thinking about that but couldn't come up with a good wording that covers resources and instrumentation scopes at the same time.